tests/acceptance/test_organization_group_index.py

```diff
@@ -69,15 +55,62 @@ def test_with_results(self, mock_now):
             },
             project_id=self.project.id
         )
-        self.browser.get(self.path)
-        self.wait_until_loaded()
-        self.browser.wait_until('.event-issue-header')
+
+    def test_with_onboarding(self):
+        self.project.update(first_event=None)
+        self.page.visit_issue_list(self.org.slug)
+        self.browser.wait_until_test_id('awaiting-events')
+        self.browser.snapshot('organization issues onboarding')
+
+    def test_with_no_results(self):
+        self.project.update(first_event=timezone.now())
+        self.page.visit_issue_list(self.org.slug)
+        self.browser.wait_until_test_id('empty-state')
+        self.browser.snapshot('organization issues no results')
+
+    @patch('django.utils.timezone.now')
+    def test_with_results(self, mock_now):
+        mock_now.return_value = datetime.utcnow().replace(tzinfo=pytz.utc)
+        self.create_issues()
+        self.page.visit_issue_list(self.org.slug)
+        self.page.wait_for_stream()
         self.browser.snapshot('organization issues with issues')
 
         groups = self.browser.find_elements_by_class_name('event-issue-header')
         assert len(groups) == 2
         assert 'oh snap' in groups[0].text
         assert 'oh no' in groups[1].text
 
-    def wait_until_loaded(self):
-        self.browser.wait_until_not('.loading')
+    @patch('django.utils.timezone.now')
+    def test_resolve_issues(self, mock_now):
+        mock_now.return_value = datetime.utcnow().replace(tzinfo=pytz.utc)
+        self.create_issues()
+        self.page.visit_issue_list(self.org.slug)
+        self.page.wait_for_stream()
+
+        self.page.select_issue(1)
+        self.page.select_issue(2)
+        self.page.resolve_issues()
+
+        self.browser.wait_until('[data-test-id="resolved-issue"]')
```

Comment:
You could make this a method of the page object. That would let the tests not have to change abstraction levels. Right now you switch between the page object mode (higher level) and direct selectors (lower level).